definitions/npm/newrelic_v4.x.x/flow_v0.53.x-/newrelic_v4.x.x.js

```diff
@@ -0,0 +1,71 @@
+// @flow strict
+/* eslint-disable flowtype/no-weak-types */
+
+// @see https://docs.newrelic.com/docs/agents/nodejs-agent/supported-features/nodejs-agent-api
+declare module 'newrelic' {
+
+  declare interface TransactionHandle {
```

Comment:
Generally we've been finding interfaces to be less performant. Do you definitely need it to be an interface?

Reply:
Oh! I wasn't aware of that. I'm moving this to a type.